Dockerfile

```
@@ -1,13 +1,14 @@
-FROM fedora:30
-RUN dnf install -y \
-    npm \
-    chromium \
-    && mkdir -p /root/.config/configstore \
-    && echo '{"isErrorReportingEnabled": false}' > /root/.config/configstore/lighthouse.json
+FROM alpine:3.12
+
+RUN apk update
+
+RUN apk add npm chromium
```

```suggestion
RUN apk -u \
 --no-cache \
 add npm chromium
```
A minor improvement here. You can do an upgrade+add, and skip baking the cache into the image with this trick.

---

ooh, TIL. Yeah, let's do it!

---

Looks like it needs `-U` or `--update-cache`

---

https://github.com/iFixit/lighthouse-docker/pull/6/commits/881a19f3c9a66a45217f5d73ea93a39189bf2b13